src/test/java/seedu/address/testutil/AssignLoaderBuilder.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.testutil;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import seedu.address.logic.commands.AssignCommand.DateTimeLoader;
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.appointment.Date;
+import seedu.address.model.appointment.Time;
+
+/**
+ * A utility class to help with building AssignLoader objects.
+ */
+public class AssignLoaderBuilder {
+    private DateTimeLoader loader;
+
+    public AssignLoaderBuilder() {
+        loader = new DateTimeLoader();
+    }
+
+    public AssignLoaderBuilder(DateTimeLoader loader) {
+        this.loader = new DateTimeLoader(loader);
+    }
```

Comment:
Should AssignLoaderBuilder be DateTimeLoaderBuilder instead? 
Because there is no AssignLoader object.

Reply:
Yes, thanks for pointing out